core/server/src/main/java/alluxio/master/file/FileSystemMaster.java

```diff
@@ -277,7 +278,7 @@ public void processJournalEntry(JournalEntry entry) throws IOException {
         long fileId = ((AsyncPersistRequestEntry) innerEntry).getFileId();
         scheduleAsyncPersistenceInternal(getPath(fileId));
       } catch (AlluxioException e) {
-        throw new RuntimeException(e);
+        LOG.error(e.getMessage());
```

Comment:
Because when the journal replays, it's possible to reschedule the async persist calls, which may end up in exceptions because the blocks may no longer be in the workers.